- Add new patchfile files/patch-doc-sam.1
   (obtained from Debian package)

--- patch-doc-sam.1 begins here ---
--- doc/sam.1	23 Nov 1997 01:40:01 -0000
+++ doc/sam.1	8 Feb 2002 16:28:04 -0000
@@ -1,35 +1,3 @@
-.de F
-.B
-.if !"\\$1"" \&\\$1 \\$2 \\$3 \\$4 \\$5 \\$6
-..
-.de L
-.B
-.if !"\\$1"" \&\\$1 \\$2 \\$3 \\$4 \\$5 \\$6
-..
-.de FR
-.BR "\\$1" "\\$2" "\\$3" "\\$4" "\\$5" "\\$6"
-..
-.de LR
-.BR "\\$1" "\\$2" "\\$3" "\\$4" "\\$5" "\\$6"
-..
-.de CW
-.ft B
-..
-.\" This is gross but it avoids relying on internal implementation details
-.\" of the -man macros.
-.de TF
-.br
-.IP "" \w'\fB\\$1\ \ \fP'u
-.PD0
-..
-.de EX
-.CW
-.nf
-..
-.de EE
-.fi
-..
-.\" delete above this point if your system has F, L, FR, LR, CW and TF macros
 .ds a \fR*\ \fP
 .TH SAM 1
 .CT 1 editor
--- patch-doc-sam.1 ends here ---
